### ISSUE TITLE
Can't add placeholder

### ISSUE THREAD

**Issue by shkomg** [aldeed/meteor-autoform-select2#19]
Hi,
I'm successfuly added options to my select2 this way
```
Template.stepOne.rendered= function (){                                                                                                                                                   
    this.$('select[name=country]').select2({placeholder:"Select country of operation"});                                                                                                  
    this.$('select[name=language]').select2({minimumResultsForSearch: -1,placeholder:"Marketplace main language"});                                                                       
    this.$('select[name=marketplaceType]').select2({minimumResultsForSearch: -1, placeholder:"Choose marketplace type"});                                                                 
};                                                                                                                                                             
```

but placeholder didn't pass trough.

I'm not sure if it is you package problem or those adding bootstrap styles. Maybe you have any suggestions on how to have placeholder on the place?

Thank you.

**Comment by aldeed** [comment]
v2 of this package is released, which uses select2 4.0, and placeholder will now work, either adding the attribute directly, or passing it in `select2Options` attribute object.

**Comment by shkomg** [comment]
Thank you!

2015-03-13 23:31 GMT+02:00 Eric Dobbertin <notifications@github.com>:

> v2 of this package is released, which uses select2 4.0, and placeholder
> will now work, either adding the attribute directly, or passing it in
> select2Options attribute object.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aldeed/meteor-autoform-select2/issues/19#issuecomment-79425233>
> .
>



-- 
Best regards

Serhiy Khvashchuk
